packages/cli/src/index.ts

```diff
@@ -837,7 +837,7 @@ yargs
             alias: 'd',
             type: 'string',
             description:
-              'Specifies the project content directory. The default value is the directory where the command is executed.',
+              'Specifies the destination folder to eject components into.',
```

Comment:
I don't think this is correct, sorry. The `project-dir` should be the directory where your actual project is, we use this across some of the new commands. It's NOT the destination directory (but then how do you say where you want that to be?)

Reply:
I updated this description based on the behavior I observed. In this implementation, this option only controls where the ejected files end up.

Reply:
Updated based on new information